build.gradle

```diff
@@ -126,7 +122,7 @@ subprojects { Project project ->
                 buildArgs.put('JAVA_VERSION', "${project.java.version.major}u${project.java.version.update}-${project.java.version.extra}");
                 buildArgs.put('JAVA_VERSION_COMPACT', "${project.java.version.major}u${project.java.version.update}${project.java.version.extra}");
             } else {
-                buildArgs.put('JAVA_VERSION', "-${project.java.version.major}.0.${project.java.version.update}+${project.java.version.extra}");
+                buildArgs.put('JAVA_VERSION', "-${project.java.version.major}.0.${project.java.version.update}+${project.java.version.extra}.1");
```

Comment:
Shouldn't the `.1` here be a part of `java.version.extra`?

Reply:
but at that point `JAVA_VERSION_COMPACT` below no longer matches ?

Reply:
needs more restructuring to actually fix that

Reply:
I don't know whether it is worth the effort to restructure. I think it was related to a rebuild, so the artifact ended up in a different download folder.
The tar.gz-file is still 11.0.9+11 and the Java version as well.
I would wait to make changes and see how the next release is

Reply:
Shouldn't these changes propagate to docker-tomcat?

Bringing java version up-to-date in:
https://github.com/xenit-eu/docker-tomcat/blob/master/tomcat-7.0/jdk-7-centos-7.gradle
https://github.com/xenit-eu/docker-tomcat/blob/master/tomcat-7.0/jdk-7-trusty.gradle
https://github.com/xenit-eu/docker-tomcat/blob/master/tomcat-7.0/jdk-8-bionic.gradle
https://github.com/xenit-eu/docker-tomcat/blob/master/tomcat-8.5/jdk-11-ubuntu-18.04.gradle

Reply:
@anghelutar Yes, I think that would be a good idea. Will create a PR in docker-tomcat.